pyproject.toml

```diff
@@ -11,6 +11,13 @@ classifiers = [
 ]
 version = "0.0.1"
 
+requires-python = ">=3.7"
```

Comment:
The README hints 3.8. Both could be correct of course. Maybe clarification?

Reply:
I would simply suggest to drop the conda lines from the readme. Would you really expect this to be able to be used in a standalone environment? If not, then those lines should definitely not be present. 

As for python version, 3.7 is very much at the end of its lifetime (numpy dropped it, astropy dropped it, a lot of other projects dropped it), for a new project I would certainly not support it.

Reply:
Removed line from the README, and changed this line to use 3.8.